components/results_table.go

```diff
@@ -301,6 +301,10 @@ func (table *ResultsTable) AppendNewRow(cells []models.CellValue, index int, UUI
 	for i, cell := range cells {
 		tableCell := tview.NewTableCell(cell.Value.(string))
 		tableCell.SetExpansion(1)
+		// Appended rows have a reference to the row UUID so we can identify them later
+		// Also, rows that have columns marked to be UPDATED will have a reference to the type of the new value (NULL, EMPTY, DEFAULT)
+		// So, the cell reference will be used to determine if the row/column is an inserted row or if it's an UPDATED row
+		// I'm sure there is a better way to do this, but it works for now
```

Comment:
Usually comment like this are not written at first person. Using the passive voice may help


```suggestion
		// there might be a better way to do this, but it works for now
```